cypress_test/integration_tests/mobile/writer/annotation_spec.js

```diff
@@ -110,9 +110,6 @@ describe('Annotation tests.', function() {
 			.should('have.text', 'some text');
 
 		mobileHelper.selectAnnotationMenuItem('Remove');
-
-		cy.get('.loleaflet-annotation-content')
-			.should('have.text', '');
```

Comment:
Here we can use something like:
```
cy.get('.loleaflet-annotation-content')
     .should('not.exist');
```
With that, we can check that the comment is actually removed. It's not a good idea just to remove an assertion here.